ms-ledger/routes/deletes.go

```diff
@@ -19,6 +19,7 @@ func (c *Controller) LedgerDelete(writer http.ResponseWriter, req *http.Request)
 		accountLedgers, err := c.GetAllLedgers()
 		if err != nil {
 			utilities.WriteStringHTTPResponse(writer, req, http.StatusInternalServerError, "Failed to retrieve all ledgers for accounts "+err.Error(), true)
+			c.lc.Errorf("Failed to retrieve all ledgers for accounts %s", err.Error())
```

Comment:
Don't repeat the message from previous line, put it in a string variable and use in both places. Same for all these log messages in all these files.